src/forms/textInput.story.jsx

```diff
@@ -2,115 +2,129 @@ import React from 'react';
 import TextInput from './TextInput';
 import Button from './Button';
 import { storiesOf } from '@kadira/storybook';
-import {
-	IntlProvider,
-	FormattedMessage,
-	defineMessages,
-} from 'react-intl';
 
 storiesOf('TextInput', module)
-	.add('default', () => (<TextInput
-		label='Your name'
-		id='fullname'
-		name='name'
-		placeholder='Not your email' />))
-	.add('with value', () => (<TextInput
-		label='Your name'
-		id='fullname'
-		name='name'
-		value='Phife Dawg'
-		placeholder='Not your email' />))
-	.add('disabled', () => (<TextInput
-		label='Your name'
-		id='fullname'
-		name='name'
-		value='Phife Dawg'
-		placeholder='Not your email'
-		disabled />))
-	.add('error state', () => (<TextInput
-		label='Your name'
-		id='fullname'
-		name='name'
-		error='Not so fast. You have an error.'
-		placeholder='Not your email' />))
-	.add('error state formatted', () => {
-		const trn = defineMessages({
-			error: {
-				defaultMessage: 'This error is a formatted message.',
-				id: 'storybook.anError',
-				description: { jira: 'SDS-204' },
-			}
-		});
+	.add('default', () => (
+			<div className='span--50'>
+				<TextInput
+					label='Your name'
+					id='fullname'
+					name='name'
+					placeholder='Not your email' />
+			</div>
+		)
+	)
+	.add('with value', () => (
+			<div className='span--50'>
+				<TextInput
+					label='Your name'
+					id='fullname'
+					name='name'
+					value='Phife Dawg'
+					placeholder='Not your email' />
+			</div>
+		)
+	)
+	.add('disabled', () => (
+			<div className='span--50'>
+				<TextInput
+					label='Your name'
+					id='fullname'
+					name='name'
+					value='Cannot focus'
+					disabled />
+			</div>
+		)
+	)
+	.add('error state', () => (
+			<div className='span--50'>
+				<TextInput
+					label='Your name'
+					id='fullname'
+					name='name'
+					error='Not so fast. You have an error.' />
+			</div>
+		)
+	)
+	.add('error state as element', () => {
 		return (
-			<IntlProvider defaultLocale='en-US' locale='en-US'>
+			<div className='span--50'>
 				<TextInput label='Your name'
 					id='fullname'
 					name='name'
-					error={<FormattedMessage {...trn.error} />}
-					placeholder='Not your email' />
-			</IntlProvider>
+					error={'<span>This error is an element</span>'}
+					placeholder='not your email' />
+			</div>
```

Comment:
You can pass the `span` as the element:
`error={<span>This is an error</span>}`